docs/backend.md

```diff
@@ -111,6 +111,25 @@ def bar(run, value: "var#foo"):
     return value * 2
 ```
 
+Pattern matching using unix shell-style
+[wildcards](https://docs.python.org/3/library/fnmatch.html) is allowed in `var#`
```

Comment:
We refer to these as glob patterns [in the EXtra-data docs](https://extra-data.readthedocs.io/en/latest/reading_files.html#extra_data.DataCollection.select) (along with a brief description). I don't feel strongly about this, though.

Reply:
sure, I just took the mention as it was from the fnmatch docs, did not know of the reference in extra-data.

updated in 847df374ee47f675fc6a70b8ed26beea4658badb